packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -190,8 +194,14 @@ export class GCFBootstrapper {
       throw Error('did not retrieve a payload from SecretManager.');
     }
     const config = JSON.parse(payload);
-    const LoggingOctokit = Octokit.plugin(LoggingOctokitPlugin);
-    return {...config, Octokit: LoggingOctokit} as Options;
+    if (logging) {
+      console.info('custom logging instance enabled');
+      const LoggingOctokit = Octokit.plugin(LoggingOctokitPlugin);
+      return {...config, Octokit: LoggingOctokit} as Options;
+    } else {
+      console.info('custom logging instance not enabled');
+      return config;
```

Comment:
I think this will need to be `return config as Options;` -- that is what it was before I added the logging part.

Reply:
Just for type safety purposes -- don't think it will make a difference at runtime